v2/stats/metrics.proto

```diff
@@ -63,6 +64,12 @@ message MemoryStat {
 	uint64 usage_limit = 33;
 	uint64 swap_usage = 34;
 	uint64 swap_limit = 35;
+    // memory.events (101-200)
```

Comment:
RFC: Is this numbering scheme right way?
cc @stevvooe

Reply:
Why wouldn't you just add them in order....???

Reply:
Because `memory.stat` will eventually have more fields.

Reply:
but does the proto numbers really matter that much to us that we have to put an upper limit on the memory fields?  Why not just add them after the events?

Reply:
added new message struct `MemoryEvents`